workbench/models.py

```diff
@@ -69,12 +74,13 @@ class XBlockState(models.Model):
     state = models.TextField(default="{}")
 
     def __repr__(self):
-        return u"<XBlockState id={self.id} " \
-            "scope={self.scope} " \
-            "scope_id={self.scope_id} " \
-            "user_id={self.user_id} " \
-            "scenario={self.scenario} " \
-            "tag={self.tag}>".format(self=self)
+        # pylint: disable=missing-format-attribute
```

Comment:
Should this warning be re-enabled at the end of the following statement?

Reply:
@jmbowman It was my understanding that this pylint disable would only be enabled for the block in which it's contained - then it would be disabled outside of the block. Is my understanding incorrect?

Reply:
That seems correct; pylint is apparently more intelligent about that than I gave it credit for.